packages/react/src/hooks/constants.ts

```diff
@@ -0,0 +1,272 @@
+import {
+  SendUserAttributeVerificationCodeOutput,
+  UpdateUserAttributesOutput,
+  UserAttributeKey,
+} from "aws-amplify/auth";
+
+export type UserAttributes = Record<UserAttributeKey, string | undefined>;
+
+export const DefaultAttributes: UserAttributes = {
+  email: undefined,
+  phone_number: undefined,
+  address: undefined,
+  birthdate: undefined,
+  email_verified: undefined,
+  family_name: undefined,
+  gender: undefined,
+  given_name: undefined,
+  locale: undefined,
+  middle_name: undefined,
+  name: undefined,
+  nickname: undefined,
+  phone_number_verified: undefined,
+  picture: undefined,
+  preferred_username: undefined,
+  profile: undefined,
+  sub: undefined,
+  updated_at: undefined,
+  website: undefined,
+  zoneinfo: undefined,
+};
+
+// Default object to be passed into useDataState
+export const defaultUpdateUserAttributesOutput: UpdateUserAttributesOutput = {
+  ["custom:placeholder"]: {
+    isUpdated: true,
+    nextStep: {
+      codeDeliveryDetails: {
+        attributeName: undefined,
+        deliveryMedium: undefined,
+        destination: undefined,
+      },
+      updateAttributeStep: "DONE",
+    },
+  },
+  address: {
+    isUpdated: true,
```

Comment:
Need clarity on why we would provide a default value that points to a successful update action, same with providing `updateAttributeStep: "DONE"` as a default value